weather.py

```diff
@@ -68,8 +92,21 @@ def find_min(weather_data):
     Returns:
         The minium value and it's position in the list.
     """
-    pass
-
+    if weather_data != []:
+        min_value = round(float(weather_data[0]),1)
+        counter = -1
+        for item in weather_data:
+            counter +=1
+            item_float = round(float(item),1)
+            if min_value >= item_float:
+                min_value = item_float
+                start_value = counter
+                item_in_type = item
+        position_value = weather_data.index(item_in_type,start_value)    
```

Comment:
Since `index` always grabs the first instance of a value, another way to approach this is by instantiating `counter = 0` and letting `position_value` = `counter`, e.g.
```suggestion
        counter = 0
        for item in weather_data:
            item_float = round(float(item),1)
            if min_value >= item_float:
                min_value = item_float
                position_value = counter
            counter +=1  
```